tools/dna_rna_tools.py

```diff
@@ -0,0 +1,141 @@
+DNA_ALPHABET = {"A", "T", "G", "C", "a", "t", "g", "c"}
+
+RNA_ALPHABET = {"A", "U", "G", "C", "a", "u", "g", "c"}
+
+RNA_COMPLEMENT_NUCLS = { "A":"U", "U":"A", "a":"u", "u":"a", "G":"C", "C":"G", "c":"g", "g":"c"}
+
+DNA_COMPLEMENT_NUCLS = {"A":"T", "T":"A", "a": "t", "t":"a", "G":"C", "C":"G", "g":"c", "c":"g"}
```

Comment:
🔥
Мне кажется только тут можно без пустых строк
```suggestion
DNA_ALPHABET = {"A", "T", "G", "C", "a", "t", "g", "c"}
RNA_ALPHABET = {"A", "U", "G", "C", "a", "u", "g", "c"}
RNA_COMPLEMENT_NUCLS = { "A":"U", "U":"A", "a":"u", "u":"a", "G":"C", "C":"G", "c":"g", "g":"c"}
DNA_COMPLEMENT_NUCLS = {"A":"T", "T":"A", "a": "t", "t":"a", "G":"C", "C":"G", "g":"c", "c":"g"}
```

Хотя это наверное зависит от настроек линтера